docs/guide/tensorboard.rst

```diff
@@ -239,7 +240,7 @@ Here is an example of how to render an episode and log the resulting video to Te
                 )
                 self.logger.record(
                     "trajectory/video",
-                    Video(th.ByteTensor([screens]), fps=40),
+                    Video(th.from_numpy(np.asarray([screens])), fps=40),
```

Comment:
```suggestion
                    Video(th.from_numpy(np.asarray([screens], dtype=np.uint8)), fps=40),
```

Thanks for the PR =)
Could you please also update the changelog? (see PR checklist)

Reply:
Thanks for your review! 
> Could you please also update the changelog? 

Sure, sorry about that, I wasn't sure whether I should still update the changelog for docs changes.

Regarding the explicit dtype, I let numpy maintain the one from the array list, as one may still be (perhaps) interested in writing out float values [0,1] https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_video.
But I don't have a strong opinion here so I would just trust your judgement.

Reply:
> in writing out float values [0,1]

I see, fine for me then, but maybe add a comment to explain what types are expected (or link to the doc).